src/main/java/net/openhft/chronicle/network/RemoteConnector.java

```diff
@@ -185,7 +185,7 @@ public boolean action() throws InvalidEventHandlerException {
 
```

Comment:
The one on line 182 ^^^ I think we can leave as debug because supposedly it only happens because of a race condition.